src/components/FaceVideo/Recording.tsx

```diff
@@ -28,22 +28,8 @@ const Recording: FunctionComponent<Props> = ({
   translate,
 }) => (
   <div>
-    <div className={style.caption}>
-      <div>
-        <div className={style.recordingIndicator}>
-          <span role="status" className={style.recordingIndicatorText}>
-            {translate('video_capture.status')}
-          </span>
-        </div>
-        {children}
-      </div>
-    </div>
+    <div className={style.caption}>{children}</div>
```

Comment:
Can we remove the outer `div`?

Reply:
Not in this case. Since the adjacent/sibling `div` for the action buttons at the next line (line 32) is still there.